docs/UserGuide.md

```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
 ### Delete task: `delete` or `dl`
 Deletes the specified task from the planner. ATHENA will ensure to leave the index of other tasks untouched.
 
-#### Format: `delete INDEX`
+#### Usage
 
-* `INDEX` refers to the index number shown in the displayed task list. It must be a positive integer.
- 
-#### Example usage
+##### Command Format
 
-* `delete 1`
-* `dl 1`
+`delete TASK-ID`
+
+##### Parameters
+The `delete` command requires 1 parameter only.
+
+* `TASK-ID` refers to the number shown beside the task that the user wants to delete in the displayed task list. 
+It must be a non-negative integer.
+
+##### Example Usage
+
+The following shows the output from ATHENA after entering `delete 1`.
+You should expect to see a message to confirm that the task is deleted. 
+ATHENA also prints the command required to restore the deleted task.
 
-#### Expected output
 
 ![delete command screenshot](screenshots/athena%20delete.jpg)
 
 
 ### View task: `view` or `v`
 Views the specified task details from the planner.
+The guide for this command is split into three sections:
 
-#### Format
-`view INDEX`
+* [Basic Usage](#basic-usage) (For users new to command line applications)
+* [Intermediate Usage](#intermediate-usage) (For users comfortable with command line applications)
 
-* `INDEX` refers to the index number shown in the displayed task list. It must be a positive integer.
- 
-#### Example of usage 
+#### Basic Usage
 
-* `view 0`
-* `v 0`
+##### Command Format
 
-#### Expected output
+`view TASK-ID`
+
+##### Parameters
+The `view` command requires 1 parameter only.
+
+* `TASK-ID` refers to the number shown beside the task that the user wants to view in the displayed task list. It must be a non-negative integer.
+
+##### Example Usage
+
+Firstly, use the [list command](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be viewed.
```

Comment:
Nice! Perhaps you could include this line under "Example Usage" of the delete command also